frameworks/remix-run/application/src/routes/$langstore.order.cart.$id.tsx

```diff
@@ -19,6 +19,6 @@ export const loader: LoaderFunction = async ({ request, params }) => {
 };
 
 export default () => {
-    const { cartId } = useLoaderData();
+    const { cartId } = useLoaderData() as { cartId: string };
```

Comment:
`useLoaderData<typeof loader>();` + change the loader using `LoaderFunctionArgs`